ocamlformat-help.txt

```diff
@@ -202,12 +202,6 @@ OPTIONS (CODE FORMATTING STYLE)
            on the preceding line and closed on the following line. The flag
            is set by default.
 
-       --escape-strings={preserve|decimal|hexadecimal}
```

Comment:
Can you add a quick CLI test to make sure we're displaying a nice error message?

Reply:
I added a test, there is no nice message for now but Jules already has a branch to work on this (https://github.com/ocaml-ppx/ocamlformat/compare/master...Julow:removed-options)

Reply:
Rebased, there is now already a test for option escape-chars.